monitoring/monitorlib/scd_automated_testing/scd_injection_api.py

```diff
@@ -0,0 +1,41 @@
+from monitoring.monitorlib.typing import ImplicitDict
+from typing import List, Literal
+from monitoring.monitorlib.scd import Volume4D
+
+SCOPE_SCD_QUALIFIER_INJECT = 'utm.inject_test_data'
+
+## Definitions around operational intent data that need to be submitted to the test injection interface
+  
+class OperationalIntentTestInjection(ImplicitDict):
+    ''' A class to hold data for operational intent data that will be submitted to the SCD testing interface. '''
+    state: str
+    priorty: int = 0
+    volumes: List[Volume4D]
+    off_nominal_volumes: List[Volume4D]= []
+
+### End of definitions around operational intent data
+
+## Definitions around flight authorisation data that need to be submitted to the test injection interface
+   
+class FlightAuthorisationData(ImplicitDict):
+    '''A class to hold information about Flight Authorisation Test '''
+    
+    uas_serial_number: str
+    operation_mode: str
+    operation_category: str
+    uas_class: str
+    identification_technologies: List[str]
+    uas_type_certificate: str
+    connectivity_methods: List[str]
+    endurance_minutes: int
+    emergency_procedure_url: str
+    operator_id: str
+    uas_id: str    
+
+### End of definitions around flight authorisation data
+
+class InjectFlightRequest(ImplicitDict):
+    ''' A class to hold the details of a test injection payload '''
+
+    operation_intent: OperationalIntentTestInjection
```

Comment:
This field name doesn't match [the API](https://github.com/interuss/dss/blob/d90153e203dd917fb57a5ab269146095567a64ab/interfaces/automated-testing/scd/scd.yaml#L418)

Reply:
Thank you! I have now corrected the typo.

Reply:
Also after moving the InjectFlightResponse there is nothing in the utils.py file, I am removing it, I will add it when we add the simulator scaffolding part.